src/internal/connector/exchange/restore_test.go

```diff
@@ -138,6 +138,7 @@ func (suite *ExchangeRestoreIntgSuite) TestRestoreEvent() {
 				ctx,
 				test.bytes,
 				suite.ac.Events(),
+				suite.ac.Events(),
```

Comment:
maybe we should make another interface that is something like `itemAndAttachmentPoster` and just pass in a single instance of the api layer. I'm not really a fan of passing in/having multiple references to the same underlying object. It seems like a good way run into unforeseen problems in the general case

Reply:
this will change significantly in an upcoming PR, and will reduce the duplicate-value population.